.github/actions/bump-golang/action.yml

```
@@ -33,7 +33,7 @@ runs:
           ref: ${{ inputs.branch }}
 
       - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@a0c478c868a71d3e239a65714de69450aa1ee2c6 # v0.76.1
+        uses: updatecli/updatecli-action@3a8785d88ec4fa03d86521a181f37c0e74627463 # v0.76.1
```

unrelated but if you don't mind:

```suggestion
        uses: updatecli/updatecli-action@3a8785d88ec4fa03d86521a181f37c0e74627463 # v2.64.0
```

This will fix the wrong version and help to know the real version in the upcoming bumps

---

ACK I thought of this too btw. I fear that we need to update it every time updatecli raises a similar PR, since it doesn't know how to update it. Maybe better to set it to `uses: updatecli/updatecli-action@v2.64.0`?

---

dependabot does change it if the version is correct. In this case, the version is not matching the sha commit. My proposal will fix the issue.

For instance, see https://github.com/elastic/oblt-actions/pull/105/files